pkg/console/subresource/consoleserver/config_builder_test.go

```diff
@@ -421,6 +422,10 @@ func TestConsoleServerCLIConfigBuilder(t *testing.T) {
 								Missing:  []authorizationv1.ResourceAttributes{{Resource: "clusterroles", Verb: "list"}},
 							},
 						},
+						PinnedResources: []operatorv1.PinnedResourceReference{
+							{Group: "apps", Version: "v1", Resource: "Deployment"},
+							{Group: "", Version: "v1", Resource: "ConfigMap"},
```

Comment:
The resource form is `deployments` and `configmaps` here, see the permissions style above